wagtail/contrib/search_promotions/tests.py

```diff
@@ -232,15 +356,29 @@ def test_results_ordering(self):
 
 
 class TestSearchPromotionsAddView(WagtailTestUtils, TestCase):
+    """
+    Test adding a search promotion.
+    """
+
     def setUp(self):
+        """
+        Login before running test.
+        """
         self.login()
 
     def test_simple(self):
+        """
+        View the add search promotions' page.
+        """
         response = self.client.get(reverse("wagtailsearchpromotions:add"))
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "wagtailsearchpromotions/add.html")
 
     def test_post(self):
+        """
+        Add search promotion with a Page and verify it appears in the
+        search promotion index page.
+        """
```

Comment:
Please remove all docstrings that are not part of new tests added.

This makes this PR have about 200 lines that are changed that have nothing to do with this PR.